src/segments/cf_target.go

```diff
@@ -0,0 +1,79 @@
+package segments
+
+import (
+	"oh-my-posh/environment"
+	"oh-my-posh/properties"
+	"regexp"
+)
+
+type CfTarget struct {
+	props properties.Properties
+	env   environment.Environment
+
+	Target CfTargetDetails
```

Comment:
If you embed it directly, you get inheritance and the template doesn't need to use the `.Target.XX` anymore. You can just use `.XX`:

```go
type CfTarget struct {
	props properties.Properties
	env   environment.Environment

	CfTargetDetails
}
```

Reply:
Noted, I'll adjust.

Reply:
Corrected.